db/JobInfo.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const jobInfoSchema = new Schema({
+  company: { type: String, required: true },
+  department: { type: String, required: true },
+  position: { type: String, required: true },
+  location: { type: String, required: true },
```

Comment:
Is Location going to be only a 'string'? If you wanna use location data have a look at this:
https://mongoosejs.com/docs/geojson.html